content/Day-6/_index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: 6. Technical Writing 
+weight: 106
+---
+
+## Technical Writing ( Day 1 )
+> _“ If everyone is moving forward together, then success takes care of itself “_ –Henry Ford
+***
+
+What is Technical Writing, it is the writing of technical content with detailed steps on how to approach or solve a problem. The importance of documentation in engineering is knowledge preservation and sharing. When members of our team are unfamiliar with a system or technology they can read and use the documentation as reference to learn more about how the system functions. Perhaps more importantly, they can form questions about the functionality of methods and design plans based on their existing knowledge. Many organizations have already documented the processes, design plans, technical architecture, logical and physical designs. Our task is to get familiar with the documentation and have our colleagues help with fill in any knowledge gaps as we get better acquainted with systems and workflows within the organization. 
+
+Documenting customer interactions and support calls is a form of documentation that can help share the knowledge of a call with other team members during a given work week. They can review our notes and discern the next logical steps to take to help resolve the issue without repeating previous steps. Writing effective technical documentation, user manuals or call notes follow the same process which includes but is not limited to:
+
+* Formatting and organizing documentation for clarity
+* Reviewing and editing documentation for accuracy and completeness
+* Knowledge of our audience (readers)
+* Visual aids 
+* Step-by-step instructions and conditional call outs 
+
+## Writers Framework 
+
+Our first steps help us create the framework for our documentations and determine what we are writing about, who we are writing for and what we want our readers to know before we start writing. These are the fundamental parts of our writing. This is a universal framework that we can use to create presentations, research papers, fiction and non-fiction artifacts. 
+
+### Writers Framework 
+1. Who needs to read this document?    
+2. When should they read this document and what should it include/exclude?
+3. What is the document about? 
+4. How can they use this information going forward?  
+5. Identify Key Points or Objectives 
+
+### Situation: Using The Writers Framework For Password Reset 
+
+**A customer password reset** password reset is the most common help desk issue. It’s also the easiest one to resolve. Let’s walk through an example of a framework. We work in a call center and we get multiple calls a day. Assume that we have a note taking application for call notes and a knowledge management system for research. You are a Technical Service Analyst and you receive a call for a password reset. 
```

Comment:
This can be an example shortcode

Reply:
This is meant to be content and context and extends into further reading after the update here